python/pyspark/ml/stat.py

```diff
@@ -323,12 +339,17 @@ def metrics(*metrics):
 
         The following metrics are accepted (case sensitive):
          - mean: a vector that contains the coefficient-wise mean.
+         - sum: a vector that contains the coefficient-wise sum.
          - variance: a vector tha contains the coefficient-wise variance.
+         - std: a vector tha contains the coefficient-wise standard deviation.
          - count: the count of all vectors seen.
+         - weightSum: the sum of all weights seen.
```

Comment:
Is this a supported metric? I don't see it in the scala code above. Same with other new values below.